src/pod-security-policy/seccompv2/src.cel

```diff
@@ -0,0 +1,122 @@
+variables:
+- name: containers
+  expression: 'has(variables.anyObject.spec.containers) ? variables.anyObject.spec.containers : []'
+- name: initContainers
+  expression: 'has(variables.anyObject.spec.initContainers) ? variables.anyObject.spec.initContainers : []'
+- name: ephemeralContainers
+  expression: 'has(variables.anyObject.spec.ephemeralContainers) ? variables.anyObject.spec.ephemeralContainers : []'
+- name: allowAllProfiles
+  expression: |
+    has(variables.params.allowedProfiles) && variables.params.allowedProfiles.exists(profile, profile == "*")
+- name: exemptImagePrefixes
+  expression: |
+    !has(variables.params.exemptImages) ? [] :
+      variables.params.exemptImages.filter(image, image.endsWith("*")).map(image, string(image).replace("*", ""))
+- name: exemptImageExplicit
+  expression: |
+    !has(variables.params.exemptImages) ? [] : 
+      variables.params.exemptImages.filter(image, !image.endsWith("*"))
+- name: exemptImages
+  expression: |
+    (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+      container.image in variables.exemptImageExplicit ||
+      variables.exemptImagePrefixes.exists(exemption, string(container.image).startsWith(exemption)))
+- name: unverifiedContainers
+  expression: |
+    (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+      !variables.allowAllProfiles &&
+      !(container.image in variables.exemptImages))
+- name: inputAllowedProfiles
+  expression: |
+    !has(variables.params.allowedProfiles) ? [] : variables.params.allowedProfiles
+- name: allowedLocalhostFiles
+  expression: |
+    has(variables.params.allowedLocalhostFiles) ? variables.params.allowedLocalhostFiles : []
+- name: allowedProfilesTranslation
+  expression: |
+    (variables.inputAllowedProfiles.filter(profile,
+    profile != "Localhost").map(profile, profile == "Unconfined" ? "unconfined" : profile)) + 
+    (variables.inputAllowedProfiles.exists(profile, profile == "RuntimeDefault") ? ["runtime/default", "docker/default"] : [])
```

Comment:
No need to do the mapping for the v2 policy, that may simplify the code a bit.

Reply:
Do you mean we should assume we have only SC style profile names in the input.params.allowedProfile since we are not using annotations in v2? 
if that is the case, then we would not need to canonicalize `obj.securityContext.seccompProfile.type` as well.

Reply:
Yep!